conanfile.py

```diff
@@ -69,6 +69,10 @@ def requirements(self):
 
     # Let people provide their own CMake for now
     # def build_requirements(self):
+    #     # nasm and strawberryperl are required for Windows builds. Uncomment when generating lockfile on non-Windows.
+    #     if self.settings.os == "Windows":
+    #         self.tool_requires("nasm/2.15.05")
+    #         self.tool_requires("strawberryperl/5.32.1.1")
```

Comment:
delete this? Transitive requirements will be added to conan.lock already if they are picked up.